packages/daimo-api/src/server/tokenRegistry.ts

```diff
@@ -0,0 +1,94 @@
+import { ForeignCoin, USDbC, assert, getHomeCoinToken } from "@daimo/common";
+import { Address, getAddress, isAddress } from "viem";
+
+import { chainConfig } from "../env";
+import { fetchWithBackoff } from "../network/fetchWithBackoff";
+
+// Exclude native ETH
+export type ForeignToken = ForeignCoin & { token: Address };
+
+type TokenList = {
+  tokens: {
+    chainId: number;
+    address: Address;
+    name: string;
+    symbol: string;
+    decimals: number;
+    logoURI?: string;
+  }[];
+  version: any;
+};
+
+// TODO: in the future, load a token registry for each chain.
```

Comment:
👍 

we can also do a single TokenRegistry that knows where coins are on each chain, tbd